### ISSUE TITLE
[FE] Profile: Resolve MVI 207 errors (Updates fail due to deprecated ID's) 

### ISSUE THREAD

**Issue by mshea0606** [department-of-veterans-affairs/va.gov-team#20509]
## Background
See ticket: [[BE] Profile: Are we sending deprecated IDs to VA Profile? #17739](https://github.com/department-of-veterans-affairs/va.gov-team/issues/17739)

Per ticket #17739 above, there is no work for FE or BE at this time.

Current status as of 2/25/21: VA Profile is triaging ticket and looking at MVI to see what's going on. No time estimate at this time.

## Tasks
- [ ] Look into why we are sending deprecated IDs to VA Profile sometimes.
- [ ] Figure out if this is a larger identity issue, and if we need to bring this to the attention of someone on the identity team.

**Comment by mshea0606** [comment]
Status as of 2/25/21: VA Profile is triaging ticket and looking at MVI to see what's going on. No time estimate at this time.

**Comment by Samara-Strauss** [comment]
3.5.21 — Next steps: This issue is going to be escalated to the VA Profile production operations (ProdOps) team. They will reach out to us over email for more information.

**Comment by Samara-Strauss** [comment]
Adding blocked label to indicate we can't move this forward without action from VA Profile team.

**Comment by Samara-Strauss** [comment]
**From VA Profile call 3.19.21**

I was told this was elevated to Josh Lindsey, who is investigating what is going on. Nothing needed from our team at this time, but Josh will likely follow-up once his investigation is complete.

**Comment by Samara-Strauss** [comment]
From today's VA Profile meeting — Josh Lindsey is supposed to be looking into this, but Mike didn't have an update on that today. He said to check in with him next week to see if he's had a chance to regroup with Josh.

**Comment by Samara-Strauss** [comment]
4.30.21 Update — Still waiting for word on this from Josh Lindsey, but no other updates on this besides that in today's VA Profile meeting. I've pinged him in Slack as well to see if I can get an answer more directly.

**Comment by Samara-Strauss** [comment]
5.4.21 update: Just had a call with Josh Lindsey, and he said in his preliminary investigation, it looks like the following is happening:

- There's an issue with ICNs.
- When they ask MPI ask for ICN information, it says they don’t exist. 
- Then MPI throws an error.

Josh said he needs Lihan's help to look into this further, but still awaiting more details on what that might look like.

**Comment by Samara-Strauss** [comment]
Latest from Josh Lindsey via Slack:

> MVI207 - It appears that the error message reported to VA.gov by VAPRO is not really an accurate status. The is symptomatic of a race condition with the identity add with MPI and usage of the identity to update contact information.
> 
> Status: Blocked
> 
> VAPRO has sent information to MPI to review for feedback. Triage on hold pending this feedback.

**Comment by Samara-Strauss** [comment]
Latest update: Josh Lindsey has documented these issues and asked for more information from MPI on what resolution would look like.

**Comment by Samara-Strauss** [comment]
Update from today's VA Profile meeting: VA Profile is working on 207 error. An update is going out Monday, so we should see a drop  in these errors after that.
